docs/applications/big-data/big-data-in-the-linode-cloud-streaming-data-processing-with-apache-storm.md

```diff
@@ -0,0 +1,1095 @@
+
+This guide will explain how to configure a working Storm cluster and its Zookeeper nodes, but it will not provide information on how to develop custom topologies for data processing. For more information on creating and deploying Storm topologies, see the [Apache Storm tutorial](https://storm.apache.org/releases/current/Tutorial.html).
+
+## Before You Begin
+
+### OS Requirements
+
+-  This article assumes that the workstation used for the initial setup of the Cluster Manager Linode is running Ubuntu 14.04 LTS or Debian 8. Other distributions and operating systems have not been tested. 
+-  After the initial setup, any SSH capable workstation can be used to log in to the Cluster Manager Linode or cluster nodes.
+-  The Cluster Manager Linode can have either Ubuntu 14.04 LTS or Debian 8 installed.
+-  A Zookeeper or Storm cluster can have either Ubuntu 14.04 LTS or Debian 8 installed on its nodes. Its distribution does not need to be the same one as the one installed on the Cluster Manager Linode.
+
+{: .note}
+>The steps in this guide and in the baSH scripts referenced require root privileges. Be sure to run the steps below as `root`. For more information on privileges, see our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+### Naming Conventions
+
+Throughout this guide, we will use the following names as examples that refer to the images and clusters we will be creating:
+
+-  `zk-image1` - Zookeeper image
+-  `zk-cluster1` - Zookeeper cluster
+-  `storm-image1` - Storm image
+-  `storm-cluster1` - Storm cluster
+
+These are the names we'll use, but you are welcome to choose your own when creating your own images and clusters. This guide will use these names in all example commands, so be sure to substitute your own names where applicable.
+
+### Get a Linode API Key
+
+Follow the steps in [Generating an API Key](/docs/platform/api/api-key) and save your key securely. It will be entered into configuration files in upcoming steps.
+
+If the key expires or is removed, remember to create a new one and update the `api_env_linode.conf` API environment configuration file on Cluster Manager Linode. This will be explained further in the next section.
+
+## Set Up the Cluster Manager
+
+The first step is setting up a central *Cluster Manager* to store details of all Storm clusters, and enable authorized users to create, manage or access those clusters. This can be a local workstation or a Linode.
+
+You can set up multiple Cluster Manager nodes if necessary to satisfy organizational or security needs. For example, two departments can each have their own Cluster Manager nodes to manage their respective clusters and keep data access restricted to their own members.
```

Comment:
@alexfornuto Yes, the removal is ok with me. There is no over-arching concept here, and another department can simply repeat the guide, that is correct.

I don't mind doing this change myself but since I'm not clear about the workflow here, do let me know if you want me to make this change or you plan to do it as a PR.